cleanlab/datalab/datalab.py

```diff
@@ -102,6 +102,7 @@ def __init__(
         data: "DatasetLike",
         label_name: Optional[str] = None,
         image_key: Optional[str] = None,
+        trained_datalab: Optional[Datalab] = None,
```

Comment:
We'll use this variable name during the development of this PR, but we should brainstorm better variable names for this argument before merging.

Reply:
Yeah, this is also what I think.